### PR TITLE
Theme - Cicada (template)

### PR DESCRIPTION
**Pull request by opnsenseuser12123**

### CODE REVIEW

**Comment by opnsenseuser12123** [comment]
Hope this works no.

**Comment by fabianfrz** [comment]
@opnsenseuser12123 I think I did restore all files you changed by accident in the theme rebellion theme.

Assigning it to @fichtner for review.

**Comment by fichtner** [comment]
I have a question: if this is based on @marjohn56's work, what makes it special? wouldn't time be spent more efficiently if you both improved the existing dark theme?

**Comment by opnsenseuser12123** [comment]
This is not based on bis Template. 
Nothing is from him




Samstag, 10. März 2018, 21:59 +0100 von notifications@github.com  <notifications@github.com>:

**Comment by opnsenseuser12123** [comment]
I made the whole Template by myself. Nothing from marjohn56




Samstag, 10. März 2018, 21:59 +0100 von notifications@github.com  <notifications@github.com>:

**Comment by marjohn56** [comment]
Nothing to do with me, I know nothing about it.

**Comment by opnsenseuser12123** [comment]
?




Samstag, 10. März 2018, 22:10 +0100 von notifications@github.com  <notifications@github.com>:

**Comment by marjohn56** [comment]
Responding to comment by @fichtner

**Comment by opnsenseuser12123** [comment]
So whats the Problem? I spent one month on figuring how i make a good Template for opnsense. And now? Whats the Problem with my Template?

**Comment by opnsenseuser12123** [comment]
I was told if i want to make a new Template i should copy the misc folder with Rebellion Plugin and modify the makefile. Of course i used the org copy of Main.css from the org Template of opnsense and not the one from marjohn56. Thats why there is a comment from marjohn56 in my Template. Im new to github so i do not know how i get rid of this comment. But the whole Template is made 100% by myself.

**Comment by opnsenseuser12123** [comment]
The Template is already finished. And there is 99% already done. The thing that are missing Cent be done behause they use javascript that cant be Managed by css. But Test it and see the difference between this and the rebellion theme.

**Comment by fichtner** [comment]
Then please fix up your PR, it's misleading.

**Comment by fabianfrz** [comment]
@fichtner it looks like this commit got into by accident. I reverted it via the webgui in 48f7742 and 8921bb1 since I can probably not push on his repository via SSH.

File: misc/theme-cicada/pkg-descr [hunk 1]
```diff
@@ -1 +1,5 @@
-The grey-orange theme.
+Cicada Theme 
+Orange Grey Dark
+(c) remic-webdesign
```
Comment (fichtner) [inline misc/theme-cicada/pkg-descr]: If you assert copyright without a license we cannot merge this, because nobody will be able to use it.

**Comment by opnsenseuser12123** [comment]
Then clear the last message in the pkg and everything is fine.




Samstag, 10. März 2018, 22:59 +0100 von notifications@github.com  <notifications@github.com>:

**Comment by opnsenseuser12123** [comment]
I say it again. I’m new to this and I thought I would be helped here instead it gets really complicated

**Comment by opnsenseuser12123** [comment]
@franco You are admin you can clear the copyright entry. That would be great.thx

**Comment by marjohn56** [comment]
@opnsenseuser12123 

There's a little more to it than that. Your commit also has the wrong path, it's not a major issue for you to correct that, and you can also correct the copyright and anything else at the same time.

Github can be a total pain, and I make mistakes even though I have been using it for quite a while. I'll try and help you, but firstly what operating system are you using? For scripts and things like themes I use windows Git Desktop, as at's quick and easy. For binaries I have to use FreeBSD. The shell commands are the same in both, but with Git Desktop in windows its really easy to create branches, commits and pushes are a doddle too.

**Comment by opnsenseuser12123** [comment]
just do not come with the shit. here is the theme in zip format. If anyone wants it, he can use it here. I do not want to deal with the stuff anymore.bye
[theme-cicada.zip](https://github.com/opnsense/plugins/files/1800134/theme-cicada.zip)

**Comment by fabianfrz** [comment]
I was able to create a build from the zip file if anyone wants to try it out:
```bash
pkg add https://files.fabian-franz.eu/os-Theme-Cicada-devel-0.1_1.txz
```

**Comment by opnsenseuser12123** [comment]
Wie kann ich das mein Repo mit meinem Repo neu syncen?

Der Fork Button macht gar nichts.

Und alles was ich über die Github Doku rausbekommen hab, funktioniert nicht. 

Weder merger usw…

Irgendetwas funktioniert nicht, weiß aber nicht was!

 

Danke nochmals 

René

 

Von: Fabian Franz BSc <notifications@github.com> 
Gesendet: Sonntag, 11. März 2018 14:10
An: opnsense/plugins <plugins@noreply.github.com>
Cc: opnsenseuser12123 <remic-webdesign@chello.at>; State change <state_change@noreply.github.com>
Betreff: Re: [opnsense/plugins] Theme - Cicada (template) (#593)

 

I was able to create a build from the zip file if anyone wants to try it out:

pkg add https://files.fabian-franz.eu/os-Theme-Cicada-devel-0.1_1.txz

**Comment by fabianfrz** [comment]
@opnsenseuser12123 you cannot fork if you already have a forked repository. I would suggest to check your history (`git log`) maybe something is broken.

**Pull request closed without merging**